server/src/instant/flags.clj

```diff
@@ -296,10 +296,10 @@
                                      [:e2e-logging :invalidator-every-n])
                              10000)))))
 
-(defn app-deletion-sweeper-disabled? []
+(defn hard-deletion-sweeper-disabled? []
   (-> (query-result)
```

Comment:
Probably safer to move this down and use `toggled?` directly, e.g.

```clj
(defn hard-deletion-sweeper-disabled? []
  (toggled? :hard-deletion-sweeper-disabled?))
```

Reply:
Oo nice! Missed this